schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
+CREATE DATABASE vet_clinic;
 CREATE TABLE animals (
-    name varchar(100)
-);
+    id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
```

Comment:
- [ ] [OPTIONAL] You can add these optional specs for the `id` field: `NOT NULL`, `AUTO INCREMENTED`